src/styles/blocks/product-card.scss

```diff
@@ -94,6 +63,37 @@
     }
   }
 
+  &__bottom-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-count {
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-text;
+    }
+  }
+
+  &__bottom-price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+
+    &-title {
```

Comment:
if code, or title you position not as a modifier write full-name selector `&__top-code` and `&__top-title`.
Or add --code or --title as modifier.

For all files*